src/gui/CMakeLists.txt

```diff
@@ -1407,6 +1407,12 @@ target_include_directories(qgis_gui SYSTEM PRIVATE
   ${QT_QTUITOOLS_INCLUDE_DIR}
 )
 
+if (WITH_QT6)
+  target_include_directories(qgis_gui SYSTEM PRIVATE
+    ${${QT_VERSION_BASE}Svg_INCLUDE_DIRS}
```

Comment:
This one shouldn't be required since it has been included publicly to qgis_core

Reply:
Hm, it is though... without it gui classes can't find QSvgWidget.h

Reply:
ok, strange!